### PR TITLE
update: making time out for a delay result in 5 second

### PR DESCRIPTION
**Pull request by arufaki**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 function increment(x, y) {
     let result = x * y;
-    console.log(result);
+    setTimeout(() => {
+        console.log(result);
+    }, 5000);
```
Comment (furradev) [inline index.js:5]: please make it longer than 5 seconds

File: index.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 function increment(x, y) {
     let result = x * y;
```
Comment (furradev) [inline index.js:2]:
```suggestion
    let result = x + y;
```

Comment (furradev) [inline index.js:2]:
```suggestion
    let result = x + y;
```

File: index.js (hunk 1 above)
Comment (arufaki) [reply index.js:5]: okay, i will fix it right now

**Review by furradev — CHANGES_REQUESTED** [review]
make the timeout longer

**Review by furradev — APPROVED** [review]
nice changes, i like your code. Thanks for collaboration btw :)

**Pull request closed without merging**